### PR TITLE
GCS bucket mover: modernize/update to py3

### PR DESCRIPTION
**Pull request by markueez**
Bringing `gcs-bucket-mover` into 2022

### CODE REVIEW

**Review by skaag — APPROVED** [review]
Looking good! Clean and better styling than before. Important PR. Thank you @markueez

**Comment by CYarros10** [comment]
I was unable to run the tool - build/import issues. tried installing source running bin/bucket_mover and tox/pex as well

**Comment by skaag** [comment]
@aymanfarhat ping 😎

File: tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_tester.py [hunk 1]
```diff
@@ -29,184 +26,190 @@
...
+  notification.create()
+
+  return bucket
 
 
 def _get_random_bucket_name():
-    """Generate a random bucket name for testing purposes"""
-    return 'bucket_mover_' + str(uuid.uuid4())
+  """Generate a random bucket name for testing purposes"""
+  return "bucket_mover_" + str(uuid.uuid4())
```
Comment (shakeebshams) [inline tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_tester.py:165]: please standardize usage of single or double quotes. I see both being used in the PR

Comment (agold-rh) [reply tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_tester.py:165]: @markueez Please address the comments and the conflict in requirements.txt, and I'll try to get this merged.

**Comment by markueez** [comment]
@agold-rh conflicts resolved and formatting should now be aligned

**Review by agold-rh — APPROVED** [review]

**Pull request merged**